sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -119,10 +125,10 @@ class RunnerIOOperation(operations.Operation):
 
   def __init__(self,
                name_context,  # type: Union[str, common.NameContext]
-               step_name,
+               step_name,  # type: Any
```

Comment:
Is this not str or Optional[str]?

Reply:
It looks like this argument isn't used at all any more.

The inheritance tree looks like this:

- `operations.Operation`: does not accept a `step_name` argument
  - `RunnerIOOperation`: calls `super`. does nothing with `step_name`
    - `DataOutputOperation`: does not override `__init__`
    - `DataInputOperation`:  calls `super`. does nothing with `step_name`